src/Orleans.TestingHost/TestClusterBuilder.cs

```diff
@@ -101,7 +101,7 @@ public TestCluster Build()
             return testCluster;
         }
 
-        private static string CreateClusterId()
+        public static string CreateClusterId()
```

Comment:
make this public in case user want to configure test cluster id explicitly, through `ClusterConfiguration`